### ISSUE TITLE
On premises RHEL server skill share

### ISSUE THREAD

**Issue by chiaberry** [cityofaustin/atd-data-tech#17840]
Frank is going to give the devs a tour of the new bells and whistles and server set up

- [ ] differences with how cron is set up
- [ ] monitoring
- [ ] `/srv` vs `/home` or `/usr`
- [ ] Signals network - `traceroute` usage to verify access
- [ ] When to use `sudo` and goals we should have about its use
- [x] Auth to GitHub
  - [x] Committing as you when you're using shared repos
  - [x] Using a passcode on your keys
- [ ] Legacy systems
  - [ ] What's tracked and what's not
- [ ] LVM2

**Comment by frankhereford** [comment]
@chiaberry 👋 

I'm reviewing issues today for the upcoming sprint, and this made me think of our conversation yesterday where I said I wasn't sure what I should share in a skill-share on the new servers. You captured a great list here, and I could fill half an hour easily with the topics above, probably an hour even. I would be ready to do that at any time, if you want to schedule this.

I realize that I suggested letting this slip in deference to the documentation effort we discussed yesterday, but with this list of topics, I now feel fine if we want to go ahead with this.
